Add  global config `enable_native_fallback` for load file op (#1148)
astronomer/astro-sdk: python-sdk/docs/configurations.rst

   [astro_sdk]
   auto_add_inlets_outlets = True

> Configuring the native fallback mechanism

Reviewer: Should we update point 3 in https://astro-sdk-python.readthedocs.io/en/stable/astro/sql/operators/load_file.html#parameters-for-native-transfer too @pankajastro ?

Author: updated 6dccda0ad378c40578138421e5c21692fde27395

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Astro SDK ``LoadFileOperator`` have a fallback mechanism to load data in database using dataframe.
> By default, We have disabled this feature. This can override at task level using ``enable_native_fallback`` param.
> This might be inconvenient for some users who do want to leverage it. Such users can set the following config to ``True`` to enable it at global level.

Reviewer: We should rephrase this:

```suggestion
The ``LoadFileOperator`` has a fallback mechanism when loading data to the database from file storage as explained in :ref:`load_file_working`.
This fallback can be configured at the task level using ``enable_native_fallback`` param.
Users can also control this setting and override the default at a global level (for all tasks) by setting the following config. Set it to ``True`` to allow falling back to "pandas" path.
```

> .. code:: ini

Reviewer: ```suggestion
.. code-block:: shell
```

>    LOAD_FILE_ENABLE_NATIVE_FALLBACK = False

Reviewer: ```suggestion
   AIRFLOW__ASTRO_SDK__LOAD_FILE_ENABLE_NATIVE_FALLBACK = False
```

or by updating Airflow's configuration

> .. code:: ini

Reviewer: ```suggestion
.. code-block:: ini
```
